### DISCUSSION TITLE
[client] Support cumulative acknowledgement at partitioned topic

### DISCUSSION THREAD

**Discussion by fxbing** [apache/pulsar#18828] [General]
**Is your feature request related to a problem? Please describe.**
Currently cumulative acknowledgement is not supported on partitioned topics.

**Describe the solution you'd like**
Do `acknowledgeCumulative()` on all partitions for partitioned topics.

**Describe alternatives you've considered**
No other alternative. Users need to implement it themselves

**Additional context**
It would be much easier for people to use if we support `acknowledgeCumulative()` at partitioned topics.

**Comment by tisonkun** [comment]
See also https://github.com/apache/pulsar/pull/6796 for more information.

It seems we can `acknowledgeCumulative` on partitioned topics but the result is to ack the specific partition.

Moved to the Discussion forum since here's no design and stale for a long time. We don't use issue track to track wishing list.
